tests/ui/traits/associated_type_bound/hrtb-associated.rs

```diff
@@ -0,0 +1,31 @@
+//@ run-pass
+//@ edition:2018
```

Comment:
```suggestion
//@ check-pass
```
why is edition:2018 required?

Reply:
You're right! I added edition:2018 since I was testing it on Godbolt, but it looks like it's not actually needed. I'll remove it.  thanks!

Reply:
also change it to `check-pass`  pleae